book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -394,10 +394,9 @@ As it turns out, the system we're missing is `TransformSystem`, and we can add i
 `TransformBundle`.
 
 ```rust,ignore
-let mut game = Application::build("./", Pong)?
-    .with_bundle(TransformBundle::new())? // Add this bundle
+let game_data = GameDataBuilder::default()
```

Comment:
Outsider question: is the builder idiom still idiomatic for Rust?

Reply:
yes